scripts/onset_zscore_connectivity_matrix.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+"""
+Compute z-score matrices from base matrices using mean and std connectivity matrices.
+"""
+
+import argparse
+import logging
+import numpy as np
+from typing import List
+
+from onsetpy.io.matrix import save_matrix, load_matrix
+from onsetpy.io.utils import (
+    add_verbose_arg,
+    add_overwrite_arg,
+    assert_inputs_exist,
+    assert_matrices_compatible,
+    assert_outputs_exist,
+)
+
+
+def calculate_z_scores(

```

Comment:
This function should be place in the core of the repo (in the onsetpy folder). You could create a new folder in onsetpy named stats.py and put this function in. Ideally, I would like a unit test you can check in io for examples. I you don't know how to do it don't worry ! I will do it ! Anyway, I forgot to do the same thing for the first script.